### ISSUE TITLE
TypeOrmHealthIndicator always returning up (with no mongodb connection)

### ISSUE THREAD

**Issue by jacob87o2** [nestjs/terminus#547]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
TypeOrmHealthIndicator always returns positive result, whether it is connected to MongoDB, or not.

options passed:
```
    {
      url: '/health',
      healthIndicators: [
        async () => this.db.pingCheck('database', { timeout: 200 }),
      ],
    }
```

health result:
```
{"status":"ok","info":{"database":{"status":"up"}},"details":{"database":{"status":"up"}}}
```

## Expected behavior
It should return error if there is no connection to the MongoDB.

## Minimal reproduction of the problem with instructions
I prepared basic example here: 
https://github.com/jacob87o2/nest/tree/terminus-typeorm-health-issue-reproduction/sample/13-mongo-typeorm

After downloading/cloning the repo, go to the folder above and:

1. Install dependencies (`npm install`)
2. Run mongodb (`docker-compose up -d`)
3. Run NestJS  (`nest start`)
4. Stop mongodb (in separate terminal - `docker-compose down`)
5. Check health endpoint (go to `http://localhost:3000/health`)

## Environment

<pre><code>
[Nest Information]
terminus version         : 6.5.5
typeorm version          : 6.2.0
common version           : 6.7.2
core version             : 6.7.2

</code></pre>

**Comment by BrunnerLivio** [comment]
Thank you for this report, I can reproduce the error. Will fix asap!

**Comment by BrunnerLivio** [comment]
Fixed in 6.5.6 :tada:

**Comment by jacob87o2** [comment]
Great, thanks!
